src/main/java/bio/terra/pdao/PrimaryDataAccess.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.pdao;
+
+import bio.terra.metadata.Study;
+
+/**
+ * In the long term, we want to make the primary data store be pluggable, supporting different implementations.
+ * This interface documents what that pluggable interface might look like. Of course, we never know until we
```

Comment:
There is a cliffhanger here :)

Reply:
I like to keep you on the edge of your seat...